src/featurs/PeoplePage/PeoplePageContent/useGetDisplayPeople.tsx

```diff
@@ -0,0 +1,92 @@
+import { useSearchParams } from 'react-router-dom';
+import { Person } from '../../../types';
+
+type CenturyRange = {
+  start: number;
+  end: number;
+};
+
+type CenturyRanges = {
+  [key: string]: CenturyRange;
+};
+
+export const useGetDisplayPeople = (people: Person[]): Person[] => {
+  const [searchParams] = useSearchParams();
+  const resultArray = [...people];
+
+  const sortParam = searchParams.get('sort');
+  const orderParam = searchParams.get('order') === 'desc' ? -1 : 1;
+  const sexParam = searchParams.get('sex');
+  const centuriesParam = searchParams.getAll('centuries');
+  const queryParam = searchParams.get('query');
+
+  const centuryRanges: CenturyRanges = {
+    16: { start: 1501, end: 1600 },
+    17: { start: 1601, end: 1700 },
+    18: { start: 1701, end: 1800 },
+    19: { start: 1801, end: 1900 },
+    20: { start: 1901, end: 2000 },
+  };
```

Comment:
the Readme.md file wasn't specific about century range but in mate example it was sorting only on birth year